pkgs/test_core/lib/src/runner/console.dart

```diff
@@ -13,7 +13,8 @@ import '../util/io.dart';
 /// An interactive console for taking user commands.
 class Console {
   /// The registered commands.
-  final _commands = <String, _Command>{};
+  final _commands = <String,
```

Comment:
Fwiw, I don't like having the record type jammed in here - I would prefer a typedef for the record type?

Reply:
Added a typedef. I tried also with a trailing comma at the inline usage site and while I think it's a bit better than this version I don't think I like it as much as the typedef.